### ISSUE TITLE
not moving when "farm pokestops"unchecked.

### ISSUE THREAD

**Issue by Aniruthan2495** [finndev/PokeBuddy#1204]
i dont want to farm pokestops but i want to catch pokemon,but the bot dosent move when the farm pokestops option is unchecked

**Comment by Rab89** [comment]
yep like it always has done for your player to move farm pokesteps has to be checked
